clusterloader2/pkg/framework/framework.go

```diff
@@ -37,18 +38,31 @@ type Framework struct {
 	automanagedNamespaceCount  int
 	clientSets                 *MultiClientSet
 	dynamicClients             *MultiDynamicClient
+	clusterConfig              *config.ClusterConfig
 }
 
+type ClusterType int
+
+const (
+	DefaultCluster ClusterType = iota
+	KubemarkRootKube
+)
+
 // NewFramework creates new framework based on given kubeconfig.
```

Comment:
Please update the doc to explain what it does and how the clusterType is used.

Reply:
I removed this enum. I wasn't intuitive. I split `NewFramework` into `NewFramework` and `NewRootFramework`.